storybook/qmlTests/tests/tst_SwapModal.qml

```diff
@@ -1250,9 +1250,8 @@ Item {
         }
 
         function test_modal_pay_input_switching_accounts() {
-            skip("flaky test")
```

Comment:
This OTOH passes fine locally but fails in CI :/

```
[2024-07-24T10:45:38.941Z] 3: FAIL!  : QmlTests::SwapModal::test_modal_pay_input_switching_accounts() Compared values are not the same
[2024-07-24T10:45:38.941Z] 3:    Actual   (): true
[2024-07-24T10:45:38.941Z] 3:    Expected (): false
```

Reply:
Ammended the commit, let's see...

Reply:
One more try...

Reply:
And one more; after which I'll just remove that commit if it fails again :laughing:

Reply:
Dropped